R.cpp

```diff
@@ -0,0 +1,16 @@
+#include <iostream>
+
+using namespace std;
+
+int main()
+{
+    long int numOfDays;
+    cin >> numOfDays;
+    int years = numOfDays/365;
+    numOfDays = numOfDays-(years*365);
```

Comment:
To avoid repeating `365` in case you want to change it in the future, you can create a `const int DAYS_IN_YEAR = 365` & use it. Same for 30.